lib/3scale/backend/configuration.rb

```diff
@@ -39,11 +39,12 @@ def parse_int(value, default)
       config.workers_logger_formatter = :text
 
       # Add configuration sections
-      config.add_section(:queues, :master_name, :sentinels, :role,
-                         :connect_timeout, :read_timeout, :write_timeout, :max_connections)
-      config.add_section(:redis, :url, :proxy, :sentinels, :role,
-                         :connect_timeout, :read_timeout, :write_timeout, :max_connections,
-                         :async)
+      config.add_section(:queues, :master_name, :username, :password, :ssl, :ssl_params, :sentinels,
+                         :sentinel_username, :sentinel_password, :role, :connect_timeout, :read_timeout, :write_timeout,
+                         :max_connections)
```

Comment:
So for Que we don't use async, correct?

Reply:
Actually we do. We send the params in the `queues` section to the redis client:

https://github.com/3scale/apisonator/blob/04e28f36fddde408c41afd960aa0240b0ccb87ea/lib/3scale/backend/queue_storage.rb#L4-L13

However, `Storage` will read `config.redis.async` to decide either using the sync or async redis client:

https://github.com/3scale/apisonator/blob/593beaba401b4322aeae779b889f3e321800a3fe/lib/3scale/backend/storage.rb#L30-L36

Reply:
Got it, thanks